libs/did-resolver-lib/src/resolver.test.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, expect, test } from '@jest/globals';
+import PeerDIDResolver from './resolver';
+
+describe('sum module', () => {
+  test('test did peer resolver', () => {
+    expect(
+      new PeerDIDResolver()
+        .resolve(
+          'did:peer:2.Ez6LSqXj3dXG5zL9bd4rUB21kDg5K6mBb5nRVArPbcAU8mX6b.Vz6Mku5fqS5Gm9iZy9nZ67uL4gJYvMMV454tmXJerWHWvp7tc.SeyJhIjpbImRpZGNvbW0vdjIiXSwiaWQiOiIjZGlkY29tbSIsInMiOiJodHRwOi8vYWxpY2UtbWVkaWF0b3IuY29tIiwidCI6ImRtIn0',
+        )
+        .then((result) => console.log(result)),
+    ).not.toBeNull();
```

Comment:
A test is not about printing things, but asserting that the resolved object is what you expect. Checking that it is not null is sufficient enough for this central component. For this case, you should for example take the output of uniresolver and check that the same is produced.

Also, you only have one test, which sounds problematic. Note that negative tests are as important as positive tests. Run the following command to monitor your test coverage when applying my recommendation to add more tests:

```bash
npm run test --workspace libs/did-resolver-lib -- --coverage
```